lib/go-tc/doc.go

```diff
@@ -157,23 +157,23 @@
 // necessarily named FooV1/FooV11 etc.) indicating the new structure. For
 // example:
 //
-//     // FooV11 represents a Foo in TO APIv1.1.
-//     //
-//     // Deprecated: TO APIv1.1 is deprecated; upgrade to FooV2.
-//     type FooV11 struct {
-//         Bar string `json:"bar"`
-//     }
-//     // FooV1 represents a Foo in the latest minor version of TO APIv1.
-//     //
-//     // Deprecated: TO APIv1 is deprecated; upgrade to FooV2.
-//     type FooV1 = FooV11
-//
-//     // FooV20 represents a Foo in TO APIv2.0.
-//     type FooV20 struct {
-//         Bar string `json:"bar"`
-//     }
-//     // FooV2 represents a Foo in the latest minor version of TO APIv2.
-//     type FooV2 = FooV20
+//	// FooV11 represents a Foo in TO APIv1.1.
+//	//
+//	// Deprecated: TO APIv1.1 is deprecated; upgrade to FooV2.
+//	type FooV11 struct {
+//	    Bar string `json:"bar"`
+//	}
+//	// FooV1 represents a Foo in the latest minor version of TO APIv1.
+//	//
+//	// Deprecated: TO APIv1 is deprecated; upgrade to FooV2.
+//	type FooV1 = FooV11
+//
+//	// FooV20 represents a Foo in TO APIv2.0.
+//	type FooV20 struct {
+//	    Bar string `json:"bar"`
+//	}
+//	// FooV2 represents a Foo in the latest minor version of TO APIv2.
+//	type FooV2 = FooV20
```

Comment:
it's interesting that `go fmt` does this, considering all of their [examples of good GoDoc comments](https://go.dev/doc/comment) use spaces for code block "indentation".